components/Button.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from "@emotion/styled";
```

Comment:
Ignore the location of this comment, but when a user clicks a category the rank should start from 1. Right now if you click a category it pulls in a synth that has #3 rank in the first spot. Lets make this be #1, #2, etc,

Reply:
-Can't comment exactly where this is in the codebase but the design for the rank should match to figma. There is a gray circle in the design. 

-Under each synth, unbold and grey the text under the synth name (similar to figma)

-Unbold category names next to each synth and "only show live projects)

Do a general check against figma for things in bold

Reply:
There should be more space between the search bar and the top of the black banner. As well as between "Explore the UMAverse"

https://www.figma.com/file/JicBZVZCFFRbOESAC3EEVc/UMA---UMAverse-v2?node-id=0%3A1

Reply:
Unbold the words right below "Explore the UMAverse"

Reply:
The TVL, TVM and % change boxes look a little too wide compared to the designs